.github/workflows/codespell.yml

```diff
@@ -0,0 +1,12 @@
+# https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+name: Check spelling with codespell
+on: [push, pull_request]
+jobs:
```

Comment:
Please put blank lines around the top-level sections, to be consistent with, eg, `code_formatting.yml`.

Reply:
As above, this file is a copy of ruff.yml. I'll change them both.

Reply:
Oh, I see.  Yeah, would be good to change both.